rts/lra/lra-proxy/api/src/main/java/io/narayana/lra/proxy/logging/lraI18NLogger.java

```diff
@@ -26,10 +26,10 @@
 
 import java.util.concurrent.ExecutionException;
 
-import org.jboss.logging.Cause;
-import org.jboss.logging.LogMessage;
-import org.jboss.logging.Message;
-import org.jboss.logging.MessageLogger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
```

Comment:
Is there a reason? Were they deprecated or something?

Reply:
yes - I got classdef errors in the swarm fraction because the version didn't match (narayana must be loading multiple versions - not sure how that works though)

Reply:
ok thanks